src/store/actions/oAuthServer.actions.js

```diff
@@ -98,6 +98,7 @@ export const confirmedSaveOAuthServer = async (dispatch, pathname, server, isEdi
     };
 
     try {
+        // eslint-disable-next-line no-unused-vars
         const response = await client.post('oauth/servers', composeRateLimit(server));
```

Comment:
We don't need eslint hack `eslint-disable-next-line` here. Just remove variable assignment 😉 :
`await client.post('oauth/servers', composeRateLimit(server));`